packages/progress-bar/docs/ProgressBar.stories.tsx

```diff
@@ -1,7 +1,9 @@
 import React from "react"
+import { Story } from "@storybook/react"
 import { CATEGORIES } from "../../../storybook/constants"
 import { figmaEmbed } from "../../../storybook/helpers/figmaEmbed"
 import { ProgressBar } from "../index"
+import { StoryWrapper } from "../../../storybook/components/StoryWrapper"
 
 export default {
   title: `${CATEGORIES.components}/ProgressBar`,
```

Comment:
This has been bugging me

```suggestion
  title: `${CATEGORIES.components}/Progress Bar`,
```

Reply:
haha so fair, I've updated